src/app/modules/news/pages/news/news.component.scss

```diff
@@ -58,29 +61,58 @@
                 grid-template-columns: 1fr 30rem;
 
                 app-sidenav {
-                    border-left: 1px solid gray;
+                    border-left: 1px solid lightgray;
                 }
 
-                section {
-                    display: grid;
-                    grid-template-columns: 1fr 1fr;
-                    gap: 3rem;
+            }
+        }
+    }
+}
 
-                    .card {
-                        width: 400px;
+@media (max-width: 1280px) {
+    :host {
+        .content-container {
+            .main-container {
+                .news-container {
+                    grid-template-columns: 1fr;
+                    grid-template-rows: 1fr auto;
+                    
+                    section {
+                        padding: 0;
+                    }
 
-                        &:nth-of-type(2n+1) {
-                            justify-self: end;
-                        }
+                    app-sidenav {
+                        border: none;
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Maybe we can just hide this sidebar on max-width 1280px, because we just repeated information abut links
![image](https://github.com/grupacosmo/cosmo-page-frontend/assets/48412587/990c0ac9-d8bb-44c5-b3e5-4a7a4836c96c)